moya/doclinker.ts

```diff
@@ -0,0 +1,192 @@
+const endpointsURL = "https://github.com/Moya/Moya/blob/master/docs/Endpoints.md"
+
+const pluginWords: Set<string> = new Set(["plugin", "plugins", "plugintype"])
+const pluginsURL = "https://github.com/Moya/Moya/blob/master/docs/Plugins.md"
+
+const providerWords: Set<string> = new Set(["provider", "providers", "moyaprovider", "rxmoyaprovider", "reactivemoyaprovider"])
+const providerURL = "https://github.com/Moya/Moya/blob/master/docs/Providers.md"
+
+const reactiveSwiftWords: Set<string> = new Set(["reactiveswift", "reactivemoya"])
+const reactiveSwiftURL = "https://github.com/Moya/Moya/blob/master/docs/ReactiveSwift.md"
+
+const rxSwiftWords: Set<string> = new Set(["rxswift, rxmoya, rx"])
+const rxSwiftURL = "https://github.com/Moya/Moya/blob/master/docs/RxSwift.md"
+
+const targetWords: Set<string> = new Set(["target", "targettype"])
+const targetsURL = "https://github.com/Moya/Moya/blob/master/docs/Targets.md"
+
+const threadingWords: Set<string> = new Set(["thread", "threading"])
+const threadingURL = "https://github.com/Moya/Moya/blob/master/docs/Threading.md"
+
+const validationWords: Set<string> = new Set(["alamofirevalidation", "validation", "validationtype", "validate"])
+const validationURL = "https://github.com/Moya/Moya/blob/master/docs/Examples/AlamofireValidation.md"
+
+const authpluginWords: Set<string> = new Set(["authplugin", "auth", "plugin", "plugintype"])
+const authpluginURL = "https://github.com/Moya/Moya/blob/master/docs/Examples/AuthPlugin.md"
+
+const customPluginWords: Set<string> = new Set(["plugin", "plugintype"])
+const customPluginURL = "https://github.com/Moya/Moya/blob/master/docs/Examples/CustomPlugin.md"
+
+const moyaErrorWords: Set<string> = new Set(["moyaerror"])
+const moyaErrorURL = "https://github.com/Moya/Moya/blob/master/docs/Examples/ErrorTypes.md"
+
+const multiTargetWords: Set<string> = new Set(["multitarget"])
+const multiTargetURL = "https://github.com/Moya/Moya/blob/master/docs/Examples/MultiTarget.md"
+
+const multipartWords: Set<string> = new Set(["multipart"])
+const multipartURL = "https://github.com/Moya/Moya/blob/master/docs/Examples/MultipartUpload.md"
+
+const encodingWords: Set<string> = new Set(["encoding"])
+const encodingURL = "https://github.com/Moya/Moya/blob/master/docs/Examples/ParameterEncoding.md"
+
+const responseWords: Set<string> = new Set(["response", "moya.response"])
+const responseURL = "https://github.com/Moya/Moya/blob/master/docs/Examples/Response.md"
+
+const migrationWords: Set<string> = new Set(["migrate", "migration"])
+const migrationURL = "https://github.com/Moya/Moya/tree/master/docs/MigrationGuides"
+
+// Helpers
+
+function createComment(body: string) {
+  schedule(async () => {
+    await danger.github.api.issues.createComment({
+      owner: repo.owner.login,
+      repo: repo.name,
+      number: issue.number,
+      body: body
+    })
+  })
+}
+
+function issueIsQuestion(): boolean {
```

Comment:
I'd personally rename this to `isIssueQuestion` but that's again just a preference.